### ISSUE TITLE
manual/javaGuide/JavaHome.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#933]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/JavaHome.md
- diffs between [2.2.0/manual/javaGuide/JavaHome.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/javaGuide/JavaHome.md) and [2.3.5/manual/javaGuide/JavaHome.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/JavaHome.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/JavaHome.md   2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/javaGuide/JavaHome.md  2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Play for Java developers

 The Java API for the Play application developers is available in the `play` package. 
@@ -25,8 +26,9 @@
 1. [[HTTP form submission and validation | JavaForms]]
     1. [[Form definitions | JavaForms]]
     1. [[Using the form template helpers | JavaFormHelpers]]
-1. [[Working with Json | JavaJsonRequests]]
-    1. [[Handling and serving Json requests | JavaJsonRequests]]
+    1. [[Protecting against CSRF | JavaCsrf]]
+1. [[Working with JSON | JavaJsonActions]]
+    1. [[Handling and serving JSON | JavaJsonActions]]
 1. [[Working with XML | JavaXmlRequests]]
     1. [[Handling and serving XML requests | JavaXmlRequests]]
 1. [[Handling file upload | JavaFileUpload]]
@@ -40,6 +42,7 @@
 1. [[Calling WebServices | JavaWS]]
     1. [[The Play WS API  | JavaWS]]
     1. [[Connect to OpenID servers | JavaOpenID]]
+    1. [[Accessing resources protected by OAuth | JavaOAuth]]
 1. [[Integrating with Akka | JavaAkka]]
     1. [[Setting up Actors and scheduling asynchronous tasks | JavaAkka]]
 1. [[Internationalization | JavaI18N]]
@@ -51,13 +54,10 @@
 1. [[Testing your application | JavaTest]]
     1. [[Writing tests | JavaTest]]
     1. [[Writing functional tests | JavaFunctionalTest]]
+1. [[Logging|JavaLogging]]
+    1. [[The Logging API|JavaLogging]]

 ## Advanced topics

 1. [[Extending Play|JavaPlugins]]
     1. [[Writing Plugins|JavaPlugins]]
-
-## Tutorials
-
-1. [[Your first application | JavaTodoList]]
-1. [[Zentask | JavaGuide1]]

```

**Comment by garbagetown** [comment]
やります :hand:

**Comment by garbagetown** [comment]
merged at #1053
